src/constructs/core/stack.test.ts

```diff
@@ -54,6 +54,19 @@ describe("The GuStack construct", () => {
     });
   });
 
+  it("should expose the repository information as a property", () => {
```

Comment:
might be good to test the negative test too, but not a blocker

Reply:
Good idea. Alas, I'm not sure this is easily done. The value is obtained from the context, or by [calling `git config --get remote.origin.url`](https://github.com/guardian/cdk/blob/001912ce5d93a90c6ab5bba9828ce9352ef92e85/src/utils/git.ts#L3-L7). The latter will always resolve to `guardian/cdk`. I'm not sure how best to fall into the `undefined` path in a test. We could mock this, but not sure its worth it. WDYT?